### PR TITLE
Convert Kingdoms::size() method to static member

### PR DESCRIPTION
**Pull request by qarkai**
Use range-based for to iterate over kingdoms.

### CODE REVIEW

File: src/fheroes2/kingdom/kingdom.h [hunk 1]
```diff
@@ -223,13 +223,12 @@ class Kingdoms
...
     void AddTributeEvents( CapturedObjects & captureobj, const uint32_t day, const int objectType );
 
-    u32 size( void ) const;
-
 private:
     friend StreamBase & operator<<( StreamBase &, const Kingdoms & );
     friend StreamBase & operator>>( StreamBase &, Kingdoms & );
 
-    Kingdom kingdoms[KINGDOMMAX + 1];
+    static constexpr uint32_t size = KINGDOMMAX + 1;
```
Comment (ihhub) [inline src/fheroes2/kingdom/kingdom.h:230]: To separate members of the class/structure from the rest of variable we usually start the name from `_` symbol.

**Review by ihhub — COMMENTED** [review]
Hi @qarkai , I left just one comment. If you think it's not worth to change then let's go with these changes.

File: src/fheroes2/kingdom/kingdom.h (hunk 1 above)
Comment (qarkai) [reply src/fheroes2/kingdom/kingdom.h:230]: Ah, right!

**Comment by ihhub** [comment]
@qarkai , thank you very much for this change!

**Pull request merged**